project/src/components/tabs-item/tabs-item.tsx

```diff
@@ -6,7 +8,7 @@ type TabsItemProps = {
 function TabsItem({className, cityName}: TabsItemProps): JSX.Element {
   return (
     <li className={className}>
-      <a className="locations__item-link tabs__item" href="#">
+      <a className={`locations__item-link tabs__item ${cityName === DEFAULT_CITY ? 'tabs__item--active' : ''}`} href="#">
```

Comment:
Для этого у нас есть пакет classnames. Поставь его в проект и мы сможем классы накидывать по условию. Для простого варианта выигрыш не очевиден, но для 2-3 уже понятно зачем. Так это своего рода дефолтное решение то давай его придерживаться
```suggestion
      <a className={cn('locations__item-link', 'tabs__item', {'tabs__item--active': cityName === DEFAULT_CITY})} href="#">
```
где cn  импорт из classnames